### PR TITLE
feat: access restriction code

### PR DESCRIPTION
**Pull request by 0xZenus**

### CODE REVIEW

File: packages/tomls/src/omnibus/reya_network.toml [hunk 1]
```diff
@@ -35,19 +35,19 @@ lotteryPotAccountId = "20092"
...
-coreRouterPackage = "reya-core:1.0.10@router"
+coreRouterPackage = "reya-core:1.0.11@router"
 coreAccountNftRouterPackage = "reya-core:1.0.1@account-nft-router"
 rUSDRouterPackage = "reya-rusd:1.0.0@router"
 tokensProxyPackage = "reya-tokens:1.0.0@proxy"
 tokensRouterPackage = "reya-tokens:1.0.0@router"
 passivePoolRouterPackage = "reya-exchange-passive-pool:1.0.4@router"
 passivePerpRouterPackage = "reya-instrument-passive-perp:1.0.16@router"
-peripheryRouterPackage = "reya-periphery:1.0.3@router"
+peripheryRouterPackage = "reya-periphery:1.0.4@router"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_network.toml:45]: need to sync this with @ab PR

File: packages/tomls/src/omnibus/reya_network.toml [hunk 2]
```diff
@@ -116,7 +116,7 @@ ae_liquidator1 = "0x89520d105a125CC6165c6685de262c42113Df9c0"
 [var.co_execution_bots]
 co_execution_bot1 = "0x0d171dFaab3440c0C88F3a07d8F3e9ffE56C609a"
 
-# TODO: confirm that this is the admin for mainnet
+# note, global pausers are also admins for this flag
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_network.toml:119]: I would suggest defining admin2, admin3, admin4, and admin5 as "<%= settings.pauser1 %>", ""<%= settings.pauser2 %>" etc

Comment (avniculae) [reply packages/tomls/src/omnibus/reya_network.toml:119]: (for better visibility)

Comment (0xZenus) [reply packages/tomls/src/omnibus/reya_network.toml:119]: ok

File: packages/tomls/src/omnibus/reya_network.toml (hunk 1 above)
Comment (0xZenus) [reply packages/tomls/src/omnibus/reya_network.toml:45]: yes yes, he'll layer in his PR on top of mine

**Review by arturbeg — APPROVED** [review]

**Pull request merged**